packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -155,13 +155,12 @@ const DropdownMenuContent = React.forwardRef((props, forwardedRef) => {
       onPointerDownOutside={composeEventHandlers(
         props.onPointerDownOutside,
         (event) => {
-          const wasTrigger = context.triggerRef.current?.contains(event.target as HTMLElement);
-
+          const targetWasTrigger = context.triggerRef.current?.contains(
+            event.target as HTMLElement
+          );
           // prevent dismissing when clicking the trigger
           // as it's already setup to close, otherwise it would close and immediately open.
-          if (wasTrigger) {
-            event.preventDefault();
-          }
+          if (targetWasTrigger) event.preventDefault();
```

Comment:
Not sure how i missed this too in previous reviews but `isTrigger` feels more appropriate here to me. Like, "is this event a trigger event, if so, stop the event from happening". The event hasn't occurred yet unless this handler runs without a `preventDefault()`.

Reply:
oh I think that probably came from here actually for consistency but looks like I went the other way around instead of just changing that single one. I'll change that one too now and we'll be consistent.